x/params/handler.go

```diff
@@ -0,0 +1,49 @@
+package params
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+)
+
+func NewProposalHandler(k Keeper) govtypes.Handler {
```

Comment:
This file is inconsistent with other `handler.go` files as those are typically for message handling not proposal handling. nbd though. Mild suggestion to make a new package `x/params/proposal` where this file and `x/params/types/proposal.go` would move to... this uniqueness merits a new package I think

Reply:
I like the notion of `proposal` staying in the `types` package though. What if we just rename this file? eg. `proposal_handler.go`?

Reply:
that sounds good

Reply:
Done!